### ISSUE TITLE
Select edge-attributes inside MetricChooser

### ISSUE THREAD

**Issue by alschmut** [MaibornWolff/codecharta#217]
Depends on #211.

Use [feature/172/visualizing-edges](https://github.com/MaibornWolff/codecharta/tree/feature/172/visualizing-edges) to checkout a branch.

## Description
Currently the user can decide to use the height metric from the MetricChooser or the Pairingrate (Checkbox inside the Settings Sidebar Edges-Panel).

- We want the user to select any of the edge-attributes directly inside the MetricChooser. Maybe separated at the end of the select-list. 
- After implementing this issue we should delete the checkbox "Use Pairingrate Height" from `edgesComponent...` and Settings-Service, Treemap-Service etc.

Expect the cc.json structure do be like:
```
{
  "projectName": "Sample Project Temporal Coupling",
  "apiVersion": "1.1",
  "nodes": []
  "edges": [
    {
      "fromNodeName": "/root/src/de/Foo.java",
      "toNodeName": "/root/de/maibornwolff/codecharta/Bar.java",
      "attributes": {
        "avgCommits": 9,
        "pairingRate": 100
      }
    },
    ...
  ]
}
```

**Comment by alschmut** [comment]
Solved and merged into [feature/172/visualizing-edges](https://github.com/MaibornWolff/codecharta/tree/feature/172/visualizing-edges)
